crates/oct-platform/e2e/fixtures.ts

```
@@ -0,0 +1,136 @@
+        process: serverProcess,
+        info: { url: `http://127.0.0.1:${port}`, workDir: '' }
+    };
+};
+
+type UiFixtures = {
+  server: { url: string };
+  workspaceServer: { url: string };
+};
+
+export const test = base.extend<UiFixtures>({
+  server: async ({ }, use, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)